lib/sensors/MoistureSensor.h

```diff
@@ -0,0 +1,20 @@
+#include "mbed.h"
+#include <I2C.h>
+
+#define I2C_Addr 0x36
+
+class MoistureSensor {
+    public:
+
+        MoistureSensor(PinName sda, PinName scl, char* data, int address = I2C_Addr);   //choose the right fucking pins
+        ~MoistureSensor();
+        bool Read_Moisture();
+
+
+    private:
+
+        PinName i2c_data;
```

Comment:
member variables should have some sort of delimiter to indicate they are. in the ROS repo, we use trailing underscores. @jetkov what's your rule for firmware repo?